js/modernizr-transitions.js

```diff
@@ -18,7 +18,7 @@
  * Modernizr tests which native CSS3 and HTML5 features are available in the
  * current UA and makes the results available to you in two ways: as properties on
  * a global `Modernizr` object, and as classes on the `<html>` element. This
- * information allows you to progressively enhance your pages with a granular level
```

Comment:
This file isn't owned by Happy Coding, it's a file for a JavaScript library.

This change doesn't hurt anything so it's totally fine, but fwiw I probably would have skipped this file.